### PR TITLE
Project 12

### PR DESCRIPTION
**Pull request by volodymyrtaras**

### CODE REVIEW

File: src/main/java/twelve/Authorization.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                .getDataStore(data);
+        GoogleAuthorizationCodeFlow flow = new GoogleAuthorizationCodeFlow
+                .Builder(HTTP_TRANSPORT, JSON_FACTORY, clientSecrets, scopes)
+                .setCredentialDataStore(datastore)
+                .build();
+        LocalServerReceiver localReceiver = new LocalServerReceiver
+                .Builder()
+                .setPort(8081)
+                .build();
+        return new AuthorizationCodeInstalledApp(flow, localReceiver)
```
Comment (Taniann) [inline src/main/java/twelve/Authorization.java]: better add blank line before return

File: src/main/java/twelve/Authorization.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import java.io.Reader;
+import java.util.List;
+
+public class Authorization {
+    public static final HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+    public static final JsonFactory JSON_FACTORY = new JacksonFactory();
+    private static final String CREDENTIALS_DIRECTORY = ".oauth-credentials";
+
+    public static Credential authorize(List<String> scopes, String data) throws Exception {
+        Reader reader = new InputStreamReader(Authorization.class.getResourceAsStream("/client_secrets.json"));
```
Comment (Taniann) [inline src/main/java/twelve/Authorization.java]: do you have this file in resources because in Pr it is absent

Comment (volodymyrtaras) [reply src/main/java/twelve/Authorization.java]: of course

Comment (volodymyrtaras) [reply src/main/java/twelve/Authorization.java]: src/main/resources

**Comment by volodymyrtaras** [comment]
@Taniann fixed.

**Pull request merged**